db/queries/scores/queries.py

```diff
@@ -0,0 +1,67 @@
+"""Queries which are performed on the `scores` table.
+
+Joins allowed.
+"""
+import json
+from typing import Dict
+from typing import List
+
+from db import db
+from db.models.score import JustScore
+from db.queries.assessment_records._helpers import derive_values_from_json
+from db.schemas import JustScoreMetadata
+from sqlalchemy import func
+from sqlalchemy import select
+from sqlalchemy.orm import defer
+
+
+def get_latest_just_score_for_application_sub_crit(
+    application_id: str, sub_criteria_id: str
+) -> Dict:
+    """get_latest_just_score_for_application_sub_crit executes a query on scores
+    which returns the most recent score for the given application_id and 
+    sub_criteria_id.
+
+    :param application_id: The stringified application UUID.
+    :param sub_criteria_id: The stringified sub_criteria UUID.
+    :return: dictionary.
+    """
+    stmt = select(JustScore).where(
+        JustScore.application_id == application_id,
+        JustScore.sub_criteria_id == sub_criteria_id
+        ).order_by(JustScore.date_created.desc())
```

Comment:
⛏️ Use a limit here so we don't select every score.

Reply:
I think we mentioned this before? scalar() will always pick a maximum of 1 element already so the limit is already applied?

Reply:
I don't think the sql query will not contain this limit.

Reply:
check the emitted sql query and see if a limit is there - if not it will be worth adding one.

Reply:
no limit emitted from the query so have added that in there now 👍🏾